test_vm/src/lib.rs

```diff
@@ -630,7 +631,7 @@ impl<'invocation, 'bs> InvocationCtx<'invocation, 'bs> {
         };
 
         // But only if we're not in read-only mode.
-        if self.read_only {
+        if self.read_only() {
```

Comment:
i moved this to use the assoc fn so when we _do_ eventually move to `ContextFlags` there will be less breaking changes

Reply:
Thanks for leaving this comment, and thanks for anticipating future changes!